package.json

```diff
@@ -3,20 +3,26 @@
   "version": "0.2.0",
   "description": "Give your elements an animated canvas background!",
   "main": "dist/index.js",
+  "types": "dist/index.d.ts",
```

Comment:
Forces typescript to use this definition file for `"main"`. You don't need it, but 🤷🏽